elasticdl/python/worker/worker.py

```diff
@@ -679,12 +669,10 @@ def _run_model_call_before_training(self, features):
         ):
             self._non_embed_vars[var.name] = var
 
-        if not self._var_created:
-            if self._use_multi_ps:
-                self.init_ps_var_partition()
-            else:
-                self.report_variable()
-            self._var_created = True
+        self._var_created = True
```

Comment:
It seems that `self._var_created` is always `True`. How about moving it between L866 and L867?
https://github.com/sql-machine-learning/elasticdl/blob/2b0078dae70aaf6e827f3bbcf6dd2579c517adea/elasticdl/python/worker/worker.py#L866-L867

Reply:
At first, `self._var_created`  is `False`. Then, `_process_minibatch` will call `_run_model_call_before_training`, and set `self._var_created` to `True`.